src/niweb/apps/noclook/management/commands/csvimport.py

```diff
@@ -0,0 +1,342 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not really sure if doing a manage command for this is the right thing... but not really sure if it is a bad way to go about it.

Normally I would have put it in a script in `src/scripts`.

Reply:
I have to say I think it worked fine. I would also have done it in a script.